### ISSUE TITLE
configure Markup Formatter

### ISSUE THREAD

**Issue by jhoblitt** [jenkinsci/puppet-jenkins#532]

**Comment by rtyler** [comment]
I'm assuming this means configuring the formatter that allows plain text versus HTML in the job descriptions and things like that?

**Comment by jhoblitt** [comment]
Indeed.  I believe this is from the ` antisamy-markup-formatter` plugin.

**Comment by cdenneen** [comment]
There is no current cli way to do this? I wouldn't mind doing an exec until the type support is added.

**Comment by jhoblitt** [comment]
Not that I am aware.
